redmine/content.md

```diff
@@ -114,7 +114,7 @@ This variable allows you to control if `rake redmine:plugins:migrate` is run on
 
 ### `REDMINE_SECRET_KEY_BASE`
 
-This variable is required when using Docker Swarm replicas to maintain session connections when being loadbalanced between containers. It will create an initial `config/secrets.yml` and set the `secret_key_base` value, which is "used by Rails to encode cookies storing session data thus preventing their tampering. Generating a new secret token invalidates all existing sessions after restart" ([session store](https://www.redmine.org/projects/redmine/wiki/RedmineInstall#Step-5-Session-store-secret-generation)). If you do not set this variable or provide a `secrets.yml` one will be generated using `rake generate_secret_token`.
+This variable is required when using loadbalanced replicas to maintain session connections. It will set the `SECRET_KEY_BASE` environment variable, which is "used by Rails to encode cookies storing session data thus preventing their tampering. Generating a new secret token invalidates all existing sessions after restart" ([session store](https://www.redmine.org/projects/redmine/wiki/RedmineInstall#Step-5-Session-store-secret-generation)). If you do not set this variable, then the `secret_key_base` value will be generated using `rake generate_secret_token`. Alternatively, you can set the `SECRET_KEY_BASE` environment variable directly.
```

Comment:
Should we explicitly recommend the upstream variable now instead? (and maybe not even mention `REDMINE_SECRET_KEY_BASE` anymore?)

Reply:
Good idea. I've adjusted it to recommend `SECRET_KEY_BASE` instead.